packages/market-matcher/src/test/TestEnvironment.ts

```diff
@@ -167,15 +172,10 @@ const deploy = async () => {
         offChainDataSource,
         logger,
         deviceTypeService: new DeviceTypeService(
-            await offChainDataSource.configurationClient.get('device-types')
+            JSON.parse((await offChainDataSource.configurationClient.get()).deviceTypes)
```

Comment:
JSON.parse should possibly moved to a configurationClient getter.

Reply:
Removed JSON parsing and stringifying, using hard types instead.